xlators/storage/posix/src/posix-inode-handle.h

```diff
@@ -94,6 +94,41 @@
         }                                                                      \
     } while (0)
 
+#define MAKE_INODE_HANDLE_DIR(rpath, this, loc, iatt_p)                        \
+    do {                                                                       \
+        if (!this->private) {                                                  \
+            op_ret = -1;                                                       \
+            gf_msg("make_inode_handle", GF_LOG_ERROR, 0,                       \
+                   P_MSG_INODE_HANDLE_CREATE,                                  \
+                   "private is NULL, fini is already called");                 \
+            break;                                                             \
+        }                                                                      \
+        if (gf_uuid_is_null(loc->gfid)) {                                      \
+            op_ret = -1;                                                       \
+            gf_msg(this->name, GF_LOG_ERROR, 0, P_MSG_INODE_HANDLE_CREATE,     \
+                   "null gfid for path %s", (loc)->path);                      \
+            break;                                                             \
+        }                                                                      \
+        errno = 0;                                                             \
+        op_ret = posix_istat(this, loc->inode, loc->gfid, NULL, iatt_p, loc);  \
+>>>>>>> posix: Resolved various reviewer comment
```

Comment:
An incomplete conflict resolution ?

Reply:
Yes recently posix_istat has been changed so i have to change the code. Earlier we do pass loc_t object in posix_itsat but now the arguments are changed .